docs/concepts/dbt.rst

```diff
@@ -0,0 +1,39 @@
+.. _dbt:
+
+data build tool (dbt)
+*********************
+
+``dbt`` is an open source, command-line tool managed by `dbtlabs`_ for generating and maintaining data transformations.
+
+``dbt`` allows engineers to transform data by writing ``SELECT`` statements that reflect business logic which ``dbt``
+materializes into tables and views that can be queried efficiently.
+
+``dbt`` also allows engineers to modularize and re-use their transformation code using "packages" that can be shared
+across projects or organizations.
+
+dbt in Aspects
+##############
+
+Aspects uses the `aspects-dbt`_ package to define the transforms used by the Aspects project. This package manages
+materialized views for data tables stored in `Clickhouse`_.
```

Comment:
It also creates views, doesn't it? @bmtcril

Reply:
It does, yes, and provides some testing as well